magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/RegistryApp.scala

```diff
@@ -60,9 +60,9 @@ object RegistryApp extends App {
   logger.info("Starting MAGDA Registry")
 
   GlobalSettings.loggingSQLAndTime = LoggingSQLAndTimeSettings(
```

Comment:
Would it be easier if we set debug logging in [application.conf](https://github.com/magda-io/magda/blob/45d5f800449aee5cc676f0c921906cd2e2f43615/magda-registry-api/src/main/resources/application.conf) ?
e.g.:
```
scalikejdbc{
    global {
        loggingSQLAndTime.enabled = true
        loggingSQLAndTime.singleLineMode = true
        loggingSQLAndTime.logLevel = DEBUG
        loggingSQLAndTime.warningEnabled = true
        loggingSQLAndTime.warningThresholdMillis = 5
        loggingSQLAndTime.warningLogLevel = warn
    }
}
```
In this way, we probably can override the log setting through runtime parameter without re-compiling the registry-api / docker image.

Reply:
😬 didn't mean to commit this but that sounds good. Although I think we can already override it, but having it in application.conf makes sense, I'll make it so 👍